src/common/schema/application.js

```diff
@@ -398,14 +398,14 @@ module.exports = {
     'dischargeType'
   ],
   dependencies: {
-    veteranGrossIncome: ['understandsFinancialDisclosure'],
-    veteranNetIncome: ['understandsFinancialDisclosure'],
-    veteranOtherIncome: ['understandsFinancialDisclosure'],
-    spouseGrossIncome: ['understandsFinancialDisclosure'],
-    spouseNetIncome: ['understandsFinancialDisclosure'],
-    spouseOtherIncome: ['understandsFinancialDisclosure'],
-    deductibleMedicalExpenses: ['understandsFinancialDisclosure'],
-    deductibleFuneralExpenses: ['understandsFinancialDisclosure'],
-    deductibleEducationExpenses: ['understandsFinancialDisclosure']
+    veteranGrossIncome: { properties: { understandsFinancialDisclosure: { 'enum': [true] } } },
+    veteranNetIncome: { properties: { understandsFinancialDisclosure: { 'enum': [true] } } },
+    veteranOtherIncome: { properties: { understandsFinancialDisclosure: { 'enum': [true] } } },
+    spouseGrossIncome: { properties: { understandsFinancialDisclosure: { 'enum': [true] } } },
+    spouseNetIncome: { properties: { understandsFinancialDisclosure: { 'enum': [true] } } },
+    spouseOtherIncome: { properties: { understandsFinancialDisclosure: { 'enum': [true] } } },
```

Comment:
@alexose "extra validation on the backend" is fine, but IMO the idea is that this schema is used for making sure frontend tests pass more than submissions hitting on the backend are valid. The backend validation is really (IMO, again) meant to prevent junk spam submissions)